mondrian-generator.js

```diff
@@ -1,44 +1,56 @@
-var canvas = document.getElementById('compositionCanvas');
-var context = canvas.getContext('2d');
-
 function clearCanvas() {
   context.clearRect(0, 0, canvas.width, canvas.height);
 }
 
-/* Code courtesy of http://stackoverflow.com/questions/12796513/html5-canvas-to-png-file*/  
+function autoSaveImg(image) {
+  const link = document.createElement('a');
+  link.href = image;
+  link.download = 'new-mondrian.jpg';
+  document.body.appendChild(link);
+  link.click();
+  document.body.removeChild(link);
+}
```

Comment:
Automates the accomplishment of an otherwise repetitive task (manual button push.) However, this is still easily toggle-able, since it's enabled by a parameter passed into the master function.